packages/connect-wallet-modal/src/components/WalletsModal/components/AcceptTermsModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Button, Modal } from '@reef-knot/ui-react';
+import styled, { css } from '@reef-knot/ui-react/styled-wrapper';
+import { helpers } from '@reef-knot/web3-react';
+import { Terms, WalletModalConnectTermsProps } from '../../Terms';
+import { CommonButtonsContainer } from '../styles';
+import { useReefKnotContext } from '@reef-knot/core-react';
+
+interface Props {
```

Comment:
It usually makes sense to also export props type if there will be any component wrapping this one

Reply:
done